apps/demos/src/app.scss

```diff
@@ -0,0 +1,30 @@
+:host {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  height: 100%;
+}
+
+// replace this with nav component as soon as it is ready - if there is one
+.dt-demos-nav {
+  display: block;
+  height: 2.75rem;
+  background-color: #242424;
```

Comment:
can we use our color $gray-something?

Reply:
Not going to touch the actual code in this PR if not needed